code-examples/quarkus/kafka-quickstart/.kubernetes/kubernetes.yml

```diff
@@ -0,0 +1,62 @@
+---
+apiVersion: v1
+kind: Service
+metadata:
+  annotations:
+    app.quarkus.io/vcs-url: https://github.com/bf2fc6cc711aee1a0c2a/guides.git
+    app.quarkus.io/build-timestamp: 2021-04-15 - 11:41:34 +0000
+    app.quarkus.io/commit-id: 8aa4280eaf888ba8078280411c6ebe026d6f098e
+  labels:
+    app.kubernetes.io/name: rhoas-kafka-quickstart-example
+    app.kubernetes.io/version: latest
+  name: rhoas-kafka-quickstart-example
+spec:
+  ports:
+  - name: http
+    port: 8080
+    targetPort: 8080
+  selector:
+    app.kubernetes.io/name: rhoas-kafka-quickstart-example
+    app.kubernetes.io/version: latest
+  type: ClusterIP
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  annotations:
+    app.quarkus.io/vcs-url: https://github.com/bf2fc6cc711aee1a0c2a/guides.git
+    app.quarkus.io/build-timestamp: 2021-04-15 - 11:41:34 +0000
+    app.quarkus.io/commit-id: 8aa4280eaf888ba8078280411c6ebe026d6f098e
+  labels:
+    app.kubernetes.io/name: rhoas-kafka-quickstart-example
+    app.kubernetes.io/version: latest
+  name: rhoas-kafka-quickstart-example
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app.kubernetes.io/name: rhoas-kafka-quickstart-example
+      app.kubernetes.io/version: latest
+  template:
+    metadata:
+      annotations:
+        app.quarkus.io/vcs-url: https://github.com/bf2fc6cc711aee1a0c2a/guides.git
+        app.quarkus.io/build-timestamp: 2021-04-15 - 11:41:34 +0000
+        app.quarkus.io/commit-id: 8aa4280eaf888ba8078280411c6ebe026d6f098e
+      labels:
+        app.kubernetes.io/name: rhoas-kafka-quickstart-example
+        app.kubernetes.io/version: latest
+    spec:
+      containers:
+      - env:
+        - name: KUBERNETES_NAMESPACE
+          valueFrom:
+            fieldRef:
+              fieldPath: metadata.namespace
+        image: quay.io/rhoas/rhoas-quarkus-kafka-quickstart:latest
```

Comment:
This is wrong - the user needs to be able to change and then build the code sample locally, and have those change be deployed to openshift. Previously we were using s2i to solve this I think.

Reply:
If this is a placeholder that gets overwritten, then use a placeholder or add a comment.

Reply:
Template isn't for user specifically, but for our own use. Template is generated by quarkus itself, so manual changes in that are not adviced